freqtrade/exchange/kraken.py

```diff
@@ -72,18 +72,18 @@ def stoploss_adjust(self, stop_loss: float, order: Dict, side: str) -> bool:
         Verify stop_loss against stoploss-order value (limit or price)
         Returns True if adjustment is necessary.
         """
-        # TODO-lev: Short support
-        return (order['type'] in ('stop-loss', 'stop-loss-limit')
-                and stop_loss > float(order['price']))
+        return (order['type'] in ('stop-loss', 'stop-loss-limit') and (
+                (side == "sell" and stop_loss > float(order['price'])) or
+                (side == "buy" and stop_loss < float(order['price']))
+                ))
 
-    @retrier(retries=0)
+    @ retrier(retries=0)
```

Comment:
i'd keep decorators written together.
While i THINK this will still be functional - it'll make it very odd to search for decorators, if we're not sticking to the existing pattern (and honestly, it simply doesn't look good)